docs/tutorial/architectural_overview.md

```diff
@@ -31,20 +31,48 @@ As we go through the tutorial, you will see that several of these components are
 
 These are the three major steps that you will see in going through this tutorial. We emphasize that these make it easy for someone new to integrate there accelerator into the system. To give the users a perspective on what they need to work on, we only need the following:
```

Comment:
There are 3 major steps here, but there are 5 subsections added, maybe these can be slightly better aligned?

Reply:
Fixed!